apps/utils/locust_tests/v2/regression_suite.py

```diff
@@ -30,29 +30,26 @@ def _(environment: Environment, **kwargs):
     global master_bene_ids
     master_bene_ids = data.load_from_parsed_opts(
         environment.parsed_options,
-        db.get_bene_ids,
-        use_table_sample=False,
+        db.get_regression_bene_ids,
         data_type_name="bene_ids",
     )
 
     global master_contract_data
     master_contract_data = data.load_from_parsed_opts(
         environment.parsed_options,
-        db.get_contract_ids,
-        use_table_sample=False,
+        db.get_regression_contract_ids,
         data_type_name="contract_data",
     )
 
     global master_hashed_mbis
     master_hashed_mbis = data.load_from_parsed_opts(
         environment.parsed_options,
-        db.get_hashed_mbis,
-        use_table_sample=False,
+        db.get_regression_hashed_mbis,
         data_type_name="hashed_mbis",
     )
 
 
-validation.set_validation_goal(validation.ValidationGoal.SLA_V2_BASELINE)
+set_comparisons_metadata_path("./config/regression_suites_compare_meta.json")
```

Comment:
Can/should we not have both goals? We can check for regressions, but should also make sur ethe SLAs arent violated too.
Edit: looking at the config json, maybe they are checked, just in the configurable checks?

Reply:
We definitely could; however, the numbers for V1 and V2 SLAs are fairly arbitrary though since I just picked numbers that were high enough so that random failures didn't happen during actual runs. 

The reason why I removed this is because, assuming the regression suite is ran how it _should_ be ran (by comparing against a previous run [baseline]), the performance metrics of the current run will be validated against a baseline to ensure performance is not noticeably worse than normal. I felt the additional, arbitrary SLA validation was redundant in the usual regression suite case.

I think if we adjusted the SLAs to be accurate (instead of just arbitrarily high) we could re-enable this validation, but as it stands I don't think there's much value added on top of the baseline validation.

Note that we still always validate that no response times are greater than 10 seconds -- this is true for every Locustfile in this project.

Reply:
I was mostly concerned that because our "failure" rate is pretty high on some metrics, we could slowly degrade and eventually slip out of SLA territory, even if the original baseline was well within them. It might make sense to avoid actively checking SLAs until we get the more accurate ones though, since like you said we don't have real numbers for now, but I think checking SLAs in these tests in general still has value once we can accurately do it.

Reply:
I agree that we should reintroduce SLA checks once we have more accurate numbers (rather than numbers that are just convenient).

Note that dhe failure threshold is high for some because they do tend to vary substantially, but those that tend to vary a lot don't seem to have much effect on more important metrics like average and median response times and requests per second. Since they vary a lot, if they hit their high failure thresholds we can be more confident that something is definitely wrong.